test/Altinn.App.Api.Tests/Altinn.App.Api.Tests.csproj

```diff
@@ -15,7 +15,7 @@
     <PackageReference Include="Moq" Version="4.20.72" />
     <PackageReference Include="Swashbuckle.AspNetCore" Version="7.2.0" />
     <PackageReference Include="xunit" Version="2.9.3" />
-    <PackageReference Include="xunit.runner.visualstudio" Version="3.0.1">
+    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.2">
```

Comment:
What happened? 😄

Reply:
I would guess that I updated to 2.8.2 at some point, and v3 was released later and updated on main.